test/arrays/elliot/removeInnerMultOpt/removeInnerMult.chpl

```diff
@@ -6,7 +6,7 @@ A = B[11..20];
 
 // reindexing when lhs is not strided does not prevent optimization
 var C => A[2..9];
-var D => A[1..10 by 2];
+//var D => A[1..10 by 2];  // commented out -- TODO EJR
```

Comment:
yes, but it definitely won't be done as part of this PR. Elliot asked me to put that comment.